pkgs/applications/graphics/wonderdraft/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv
+, lib
+, requireFile
+, dpkg
+, xorg
+, libGL
+, alsa-lib
+, pulseaudio
+, fetchurl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "wonderdraft";
+  version = "1.1.7.3";
+
+  src = requireFile {
+    name = "Wonderdraft-${version}-Linux64.deb";
+    url = "https://wonderdraft.net/";
+    sha256 = "157himdhpijcc7dk696p6d38kdsk8x44jzagraa8ax1mkhbikilb";
+  };
+  sourceRoot = ".";
+  unpackCmd = "${dpkg}/bin/dpkg-deb -x $curSrc .";
+
+  dontConfigure = true;
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir -p $out/bin
```

Comment:
I forgot to say that this needs `runHook preInstall` at the beginning and `runHook postInstall` at the end. After that, its fine.